Build/Projects/FrameworkReferences.definition

```diff
@@ -25,6 +29,7 @@
         Name="OpenTK"
         Path="ThirdParty/Dependencies/OpenTK.dll" />
     </Service>
+    <Reference Include="MonoGame.Framework.Core.Include" />
```

Comment:
@dellis1972 to clear one thing up.  This change is to move non-platform specific code to it's own PCL project, this will result in all projects (except Linux/Web/WP8) to have two references per project:
* MonoGame.Framework.Core (core PCL)
* MonoGame.Framework

The Core PCL can also then be used by frameworks and projects that only need the base types from MG instead of the entire lib. (such as greatly requested by the likes of Farseer and others), this means addon frameworks will have an easier time supporting MG (and they don't need to platformise their projects)

For the Bait and switch PCL lib, that will be a separate undertaking to enable consumers to build games in a PCL project. This is different to this change, that comes later.  The added advantage of the above is that it eases this process needing less of the framework to be faked in a PCL.